### DISCUSSION TITLE
Transparent bg and one-sided lightbulb

### DISCUSSION THREAD

**Discussion by NesterenkoDanil** [nvimdev/lspsaga.nvim#1014] [Q&A] [answered]
Hi! Is there a way to make this ugly black bg transparent? Also, if it's possible, can I get a lightbulb only on the right side, since the left lightbulb pushes code to the right?
![image](https://user-images.githubusercontent.com/128042035/230981722-34844295-3893-4f87-82ec-8dbb4518d776.png)

**Comment by glepnir** [comment, marked as answer]
`lightbulb.virtual_text = false` . `hi SagaNormal guibg=NONE`

**Reply by NesterenkoDanil** [reply]
how do i set guibg=NONE?
